labelbox/data/serialization/ndjson/label.py

```diff
@@ -92,10 +93,12 @@ def _create_video_annotations(
 
         video_annotations = defaultdict(list)
         for annot in label.annotations:
+            # print(dict(annot))
```

Comment:
**suggestion** think this may be extra line we can cut out

Reply:
leftover